src/core/server/saved_objects/import/check_conflicts.ts

```diff
@@ -44,6 +44,7 @@ interface CheckConflictsParams {
   ignoreRegularConflicts?: boolean;
   retries?: SavedObjectsImportRetry[];
   createNewCopies?: boolean;
+  dataSourceId?: string;
```

Comment:
Looks like dataSourceId is never used nor passed in to the function

Reply:
I will remove it